state/carmen.go

```diff
@@ -104,6 +106,11 @@ func (s *carmenStateDB) CreateAccount(addr common.Address) {
 	s.txCtx.CreateAccount(carmen.Address(addr))
 }
 
+func (s *carmenStateDB) CreateContract(addr common.Address) {
+	panic("CreateContract not implemented")
+	return
+}
```

Comment:
Does this need to be implemented soon? Should we create an issue?

Reply:
Please see #1119